source/documentors/references/templates/doc_concept_template.rst

```diff
@@ -0,0 +1,66 @@
+Concept Document Template
+#########################
+
+A concept is a document that clarifies and illuminates a particular topic. It is understanding-oriented. It could be considered a conceptual guide. This type of document answers a why question. It could be descriptive, historical, or even propose different alternatives to explain the bigger picture and give context. 
+
+For example, “What is an XBlock? The Open edX platform provides different components, called XBlocks, that can work to create a course, like text, video, assessment, and discussions.”  (Replace this text with your content.)
+
+Since this type of document seeks to explain a topic, you will likely need to add images to exemplify the topic you are talking about. For example:
```

Comment:
```suggestion
Since this type of document seeks to explain a topic, you will likely need to add images to illustrate the topic you are talking about. For example:
```

"exemplify" is a more complicated english word than "illustrate"